subaru_crosstrek_2018.dbc

```diff
@@ -37,54 +37,52 @@ BU_: XXX X
 
 
 BO_ 2 Steering: 8 XXX
+ SG_ Steering_Angle : 7|16@0- (0.1,0) [0|65535] "" XXX
  SG_ Counter : 25|3@1+ (1,0) [0|7] "" XXX
  SG_ CHECKSUM : 32|8@1+ (1,0) [0|255] "" XXX
- SG_ Steering_Angle : 7|16@0- (0.1,0) [0|65535] "" XXX
 
 BO_ 64 Throttle: 8 XXX
  SG_ CHECKSUM : 0|8@1+ (1,0) [0|255] "" XXX
  SG_ Counter : 8|4@1+ (1,0) [0|255] "" XXX
- SG_ NEW_SIGNAL_1 : 12|4@1+ (1,0) [0|1] "" XXX
  SG_ Engine_RPM : 16|12@1+ (1,0) [0|255] "" XXX
- SG_ NEW_SIGNAL_4 : 28|5@1+ (1,0) [0|1] "" XXX
  SG_ Throttle_Pedal : 32|8@1+ (1,0) [0|255] "" XXX
  SG_ Throttle_Cruise : 40|8@1+ (1,0) [0|255] "" XXX
- SG_ Throttle_Combo : 48|7@1+ (1,0) [0|1] "" XXX
- SG_ Signal1 : 55|8@1+ (1,0) [0|255] "" XXX
- SG_ NEW_SIGNAL_3 : 56|4@1+ (1,0) [0|255] "" XXX
+ SG_ Throttle_Combo : 48|8@1+ (1,0) [0|1] "" XXX
+ SG_ Signal1 : 56|4@1+ (1,0) [0|255] "" XXX
  SG_ Off_Accel : 60|4@1+ (1,0) [0|7] "" XXX
 
-BO_ 65 NEW_MSG_1: 8 XXX
+BO_ 72 Transmission: 8 XXX
  SG_ CHECKSUM : 0|8@1+ (1,0) [0|255] "" XXX
  SG_ Counter : 8|4@1+ (1,0) [0|15] "" XXX
- SG_ NEW_SIGNAL_4 : 32|12@1+ (1,0) [0|255] "" XXX
- SG_ NEW_SIGNAL_3 : 16|12@1+ (1,0) [0|1] "" XXX
- SG_ NEW_SIGNAL_5 : 31|1@0+ (1,0) [0|15] "" XXX
- SG_ NEW_SIGNAL_6 : 48|8@1+ (1,0) [0|63] "" XXX
- SG_ NEW_SIGNAL_7 : 59|2@0+ (1,0) [0|255] "" XXX
+ SG_ Gear : 24|8@1+ (1,0) [0|255] "" XXX
+ SG_ RPM : 40|16@1+ (1,0) [0|255] "" XXX
 
-BO_ 72 Transmission: 8 XXX
+BO_ 73 CVT: 8 XXX
  SG_ CHECKSUM : 0|8@1+ (1,0) [0|255] "" XXX
  SG_ Counter : 8|4@1+ (1,0) [0|15] "" XXX
- SG_ NEW_SIGNAL_5 : 16|8@1+ (1,0) [0|255] "" XXX
- SG_ Gear : 24|8@1+ (1,0) [0|255] "" XXX
- SG_ NEW_SIGNAL_4 : 38|3@0+ (1,0) [0|255] "" XXX
- SG_ NEW_SIGNAL_3 : 40|16@1+ (1,0) [0|255] "" XXX
+ SG_ CVT_Gear : 24|8@1+ (1,0) [0|127] "" XXX
 
-BO_ 316 NEW_MSG_3: 8 XXX
+BO_ 316 Brake_Status: 8 XXX
+ SG_ CHECKSUM : 0|8@1+ (1,0) [0|1] "" XXX
+ SG_ Counter : 8|4@1+ (1,0) [0|1] "" XXX
+ SG_ Signal1 : 12|46@1+ (1,0) [0|1] "" XXX
```

Comment:
What are all these `SignalX` definitions for?

Reply:
They are filler signals for messages which are copied in carstate and rewritten in subarucan, eg https://github.com/martinl/openpilot/blob/feature-subaru-long/selfdrive/car/subaru/subarucan.py